### PR TITLE
Convert docs from MarkDown to AsciiDoc

### PR DESCRIPTION
**Pull request by fao89**
Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1098
Preview: https://logserver.rdoproject.org/04/304/65cfd5f720e8c6f34dbcf2a82bbfe1672d7bcb4c/github-check/ansibleee-operator-docs-preview/14c8a79/docs_build/docs_build/ansibleee/index-upstream.html

### CODE REVIEW

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -11,13 +19,30 @@ jobs:
...
+        run: make docs
+
+      - name: Prepare gh-pages branch
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
-          mkdocs gh-deploy --force --clean --verbose
+
+          git branch -D gh-pages &>/dev/null || true
+          git checkout -b gh-pages 95fa4ca0ec7a38ee8dc0aabcd39cbfa98745d3d4
```
Comment (rebtoor) [inline .github/workflows/docs.yaml:37]: This command will be removed/adapted after a successfull check?

File: .gitignore [hunk 2]
```diff
@@ -42,3 +42,7 @@ go.work.sum
 
 # docs
 site/
+/.bundle
+/docs_build
```
Comment (rebtoor) [inline .gitignore:46]: use a `/` after folder names

File: .github/workflows/docs.yaml (hunk 1 above)
Comment (fao89) [reply .github/workflows/docs.yaml:37]: the idea is to get only static html stuff on gh-pages branch, I'm copying the adoption logic: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/.github/workflows/docs.yaml#L38

Comment (rebtoor) [reply .github/workflows/docs.yaml:37]: ack!

**Review by rebtoor — COMMENTED** [review]
+2 for me

File: zuul.d/jobs.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+- job:
+    name: ansibleee-operator-docs-preview
+    parent: cifmw-doc
+    vars:
+      asciidoc_available: true
```
Comment (fao89) [inline zuul.d/jobs.yaml:6]: https://github.com/openstack-k8s-operators/ci-framework/pull/1098

**Pull request merged**